README.md

```diff
@@ -4,10 +4,16 @@
 </p>
 
 <h3 align="center">
-  Lightning-Fast Frontend Platform for Headless Commerce
+  <strong>An ecosystem of developer tools that accelerate and simplify building eCommerce Storefronts</strong>
```

Comment:
There's some inconsistencies in this doc between eCommerce and e-commerce.

Reply:
my bad, it should be eCommerce everywhere